src/Apps/Artwork/Utils/FilterLocations.tsx

```diff
@@ -2,7 +2,7 @@
  * Helper function to filter out blank and duplicate location names
  * @param locations collection of location objects to filter
  */
-export const filterLocations = (locations: ReadonlyArray<{ city: string }>) => {
+export const FilterLocations = (locations: ReadonlyArray<{ city: string }>) => {
```

Comment:
CapsCase typically denotes classes. For simple functions `filterLocations` is sufficient.

Reply:
Ah, k. For some reason in a fileName that looked off to me. Will update in a sec.